### ISSUE TITLE
support alt text on publication objects lead images: 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#339]
The custom "publication" content type includes a lead image. We need to make changes for accessibility support, mirroring what we've already done for the standard "Lead Image" and for the News item.

## New Alt text property

 - Add an "alt text" field to the content type so that users can add and store the appropriate text. This field should be **required** and it cannot be zero-length.

## HTML markup changes

 - suppress the ```title``` attribute on the ```<img>``` entirely
 - fill the ```alt``` attribute from the new "alt text" field on the object instead of from the item title

**Comment by cguardia** [comment]
This is now on staging.

**Comment by paregorios** [comment]
This looks great! Please merge to master and deploy to production.
